### ISSUE TITLE
Incorrectly added `-lc++` flag to `Tests` scheme

### ISSUE THREAD

**Issue by jhen0409** [rnpm/rnpm-plugin-upgrade#2]
I'm using this plugin to upgrade xcodeproj file, but it seems will add `-lc++` flag to Tests scheme.

[Use `rnpm upgrade`, it will add `lc++` to Tests scheme.](https://github.com/jhen0409/react-native-boilerplate/commit/a287d0613a63990851c08258d6409860c6594d80#diff-05462836d6767ed015e820ce62a9df78R592)
[Use `react-native upgrade`, It will add `lc++` to right place.](https://github.com/jhen0409/react-native-boilerplate/commit/9604d025ef5c54e37e3c64aa3a1f3117b1ee69aa)

**Comment by Kureev** [comment]
cc @grabbou

**Comment by grabbou** [comment]
Yeah, good catch! In fact we have discarded `rnpm upgrade` in the meantime as it turns out the `react-native upgrade` does all these things for us! 

I don't think this is harmful side-effect, esp. that you have to add it anyway when using SnapshotTests or other React-related code in that target.

Sorry for the inconvenience!
